flink-table/flink-table-common/src/main/java/org/apache/flink/table/connector/ChangelogMode.java

```diff
@@ -43,6 +43,15 @@ public final class ChangelogMode {
                     .addContainedKind(RowKind.INSERT)
                     .addContainedKind(RowKind.UPDATE_AFTER)
                     .addContainedKind(RowKind.DELETE)
+                    .deletesOnKey(true)
```

Comment:
I am curious what you think. The config here has .deletesOnKey. But all the way through the code we are using withFullDeletes as the boolean. So  deletesOnKey= true in the config is tracked as withFullDeletes = false and vice versa.  
I wonder if it is worth changing the config so it is deletesOnRecord. Then true in the config maps to withFullDeletes and both would be the same boolean.

Reply:
I think after the last update the comment is void.